grafana/overlays/moc/smaug/dashboards/thoth-station/prod/common-patch.yaml

```
@@ -0,0 +1,3 @@
+- op: add
+  path: /spec/customFolderName
+  value: Thoth-Station
```

please renmae this to 
```suggestion
  value: Thoth-Station-Prod
```

As the same folder would cause collision between moc and prod dashboards.